Doc/howto/regex.rst

```diff
@@ -526,7 +522,7 @@ of each one.
 +=================================+============================================+
 | :const:`ASCII`, :const:`A`      | Makes several escapes like ``\w``, ``\b``, |
 |                                 | ``\s`` and ``\d`` match only on ASCII      |
-|                                 | characters with the respective property.   |
+|                                 | characters with the respective property    |
```

Comment:
I guess this is done for consistency?  Any reason these shouldn't be complete sentences?  I'd have rather added missing periods instead of removed them.  Does this make the diff smaller?  Is that worth it?

Reply:
No other reasons besides consistency.

Reply:
If it is single sentences in a table, I would normally not have a period, but that's just me.